packages/react/src/components/AudioPlayer/AudioPlayerProgressBar/AudioPlayerProgressBar.tsx

```diff
@@ -0,0 +1,373 @@
+import { useEffect, useRef, useState } from 'react';
+
+import { AudioPlayerProgressBarProps } from './AudioPlayerProgressBar.types';
+
+import clsx from 'clsx';
+import { getAudioPlayerProgressBarUtilityClass } from './AudioPlayerProgressBar.classes';
+
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+
+import { styled, useThemeProps } from '@mui/material/styles';
+import { Slider, sliderClasses, Tooltip, tooltipClasses, TooltipProps } from '@mui/material';
+
+import { useWindowEventListener } from '../../../hooks';
+
+import { Instance } from '@popperjs/core';
+
+type AudioPlayerProgressBarOwnerState = {
+  classes?: AudioPlayerProgressBarProps['classes'];
+  variant?: AudioPlayerProgressBarProps['variant'];
+  isPlaying: boolean;
+  isOverTrack: boolean;
+  isOverThumb: boolean;
+};
+
+const useUtilityClasses = (ownerState: AudioPlayerProgressBarOwnerState) => {
+  const { classes } = ownerState;
+
+  const slots = {
+    root: ['root'],
+    tooltip: ['tooltip'],
+    slider: ['slider']
+  };
+
+  return composeClasses(slots, getAudioPlayerProgressBarUtilityClass, classes);
+};
+
+const AudioPlayerProgressBarRoot = styled('div', {
+  name: 'ESAudioPlayerProgressBar',
+  slot: 'Root',
+  overridesResolver: (props, styles) => styles.current
```

Comment:
```suggestion
  overridesResolver: (props, styles) => styles.root
```
Пройдись по всем, проверь что slot совпадает с overridesResolver